<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RubricCreatorGUI</class>
 <widget class="QWidget" name="RubricCreatorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RubricCreatorGUI</string>
  </property>
  <widget class="QListWidget" name="rubricSectionList">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>59</y>
     <width>239</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalPoints">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="pointValueList">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>56</y>
     <width>81</width>
     <height>254</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="addRubricSectionText">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>30</y>
     <width>239</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>189</height>
    </size>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pointValueSpinBox">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pointValueLabel">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>11</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="addRubricSectionLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add a rubric section:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>77</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="colorDropDown">
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>purple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pink</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="colorList"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="totalPointsLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Points:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>115</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="addSectionButton">
      <property name="text">
       <string>Add Section</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeSelectedButton">
      <property name="text">
       <string>Remove Selected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="doneButton">
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
